logRootQA.py

```diff
@@ -193,24 +193,29 @@ def summaryJR(jrDir):
     nDiff=0
     print(jrDir)
     dirs=[]
-    for root, dirs, files in os.walk(jrDir):
+    #find directories at top level
+    for root, dirs, _ in os.walk(jrDir):
         break
 
     nAll=0
     nOK=0
-    for d in dirs:
-        diffs=getFiles(root+'/'+d,'*.png')
+    for d, subdir, files in os.walk(jrDir):
+        if not d.split('/')[-1].startswith('all_'): continue
+        if not '_' in d: continue
+        relative_d = d.replace(root,'')
```

Comment:
this should take care of printing the directory name relative to a root that gets changed consistently when copying to the web area